test/hotspot/jtreg/ProblemList-generational-zgc.txt

```diff
@@ -105,6 +105,7 @@ serviceability/sa/jmap-hprof/JMapHProfLargeHeapProc.java      8307393   generic-
 serviceability/sa/jmap-hprof/JMapHProfLargeHeapTest.java      8307393   generic-all
 serviceability/sa/sadebugd/ClhsdbAttachToDebugServer.java     8307393   generic-all
 serviceability/sa/sadebugd/ClhsdbTestConnectArgument.java     8307393   generic-all
+serviceability/sa/ClhsdbTestAllocationMerge.java              8307393   generic-all
```

Comment:
Do you need to add TestDebugInfoDecode.java?

Reply:
No.  ClhsdbTestAllocationMerge is using `jstack -v` which doesn't work with ZGC because the `-v` part needs to iterate the heap looking for AbstractQueuedSynchronizers which doesn't work with ZGC and the SA.

Reply:
It's not -v that is making it do that. It does it without -v also. It can be a very slow operation, and I recently was running into cases where even after 10 minutes it was not done, so I gave up waiting. I'm about to file a CR to make it so it does not do that by default, and add a -l argument to optionally enable it. This will make it consistent with what bin/jstack is doing. I already have the fix ready. Just need to get the CR filed and and publish the PR.

Reply:
Ok.  I was assuming that the `-v` option was like the `-l` option.  Anyway, TestDebugInfoDecode doesn't use jstack so it doesn't need the problem list entry.

Reply:
I think generally speaking we problem list all SA tests with generational ZGC, but probably those that don't deal with the heap at all (which means they don't even use jstack) are ok.